Microsoft.HealthVault/Connection/SessionCredentialClientBase.cs

```diff
@@ -59,6 +59,8 @@ private SessionCredential GetSessionCredential(HealthServiceResponseData respons
             }
 
             SessionCredential sessionCredential = this.GetAuthenticationToken(responseData.InfoNavigator);
+            // Based on access token default lifetime: https://docs.microsoft.com/en-us/azure/active-directory/active-directory-configurable-token-lifetimes
```

Comment:
End goal is yes, we want the server to specify this. That change will take time, I'll create some tasks to get this done

---
In reply to: [115805648](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115805648) [](ancestors = 115805648,115798063)